### DISCUSSION TITLE
Search through non-project files

### DISCUSSION THREAD

**Discussion by aleksijuvani** [zed-industries/zed#23929] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

I often want to search through files that are not part of the current project, e.g. I want to search through all the header files in the Windows SDK, which is located at `C:\Program Files (x86)\Windows Kits\10\Include\10.0.20348.0\um`. The editor seems to only provide a way to search files that are a part of the current project. Adding this folder to the search include filter does not work.

**Comment by notpeter** [comment]
As a workaround you can temporarily add those folders to your project and then the find in files will search through them. 

Conceivably, we could support "including" additional directories in search beyond project dirs -- if the include item was a absolute paths (begin with `/` on mac/linux or `^([A-z]:)?\\` on windows) or home dir relative paths (`~`) -- we could expand the search path for this search.

**Comment by blushingpenguin** [comment]
This is the one missing feature that frequently drives me back to vscode
